src/main/java/com/czertainly/api/clients/NotificationInstanceApiClient.java

```diff
@@ -0,0 +1,104 @@
+        WebClient.RequestBodyUriSpec request = prepareRequest(HttpMethod.GET, connector, true);
+
+        return processRequest(r -> r.uri(connector.getUrl() + NOTIFICATION_INSTANCE_IDENTIFIED_CONTEXT, uuid)
+                .retrieve()
+                .toEntity(NotificationProviderInstanceDto.class)
+                .block()
+                .getBody(), request, connector);
+    }
+
+    public NotificationProviderInstanceDto createNotificationInstance(ConnectorDto connector, NotificationProviderInstanceRequestDto requestDto) throws ConnectorException {
+        WebClient.RequestBodyUriSpec request = prepareRequest(HttpMethod.POST, connector, true);
+
+        return processRequest(r -> r.uri(connector.getUrl() + NOTIFICATION_INSTANCE_BASE_CONTEXT)
+                .body(Mono.just(requestDto), NotificationProviderInstanceRequestDto.class)
+                .retrieve()
+                .toEntity(NotificationProviderInstanceDto.class)
+                .block()
+                .getBody(), request, connector);
+    }
+
+
+    public NotificationProviderInstanceDto updateNotificationInstance(ConnectorDto connector, String uuid, NotificationProviderInstanceRequestDto requestDto) throws ConnectorException {
+        WebClient.RequestBodyUriSpec request = prepareRequest(HttpMethod.POST, connector, true);
+
+        return processRequest(r -> r.uri(connector.getUrl() + NOTIFICATION_INSTANCE_IDENTIFIED_CONTEXT, uuid)
+                .body(Mono.just(requestDto), NotificationProviderInstanceRequestDto.class)
+                .retrieve()
+                .toEntity(NotificationProviderInstanceDto.class)
+                .block()
+                .getBody(), request, connector);
+    }
+
+    public void removeNotificationInstance(ConnectorDto connector, String uuid) throws ConnectorException {
+        WebClient.RequestBodyUriSpec request = prepareRequest(HttpMethod.DELETE, connector, true);
+
+        processRequest(r -> r.uri(connector.getUrl() + NOTIFICATION_INSTANCE_IDENTIFIED_CONTEXT, uuid)
+                .retrieve()
+                .toEntity(Void.class)
+                .block()
+                .getBody(), request, connector);
+    }
+
+
+    public void sendNotification(ConnectorDto connector, String uuid, NotificationProviderNotifyRequestDto requestDto) throws ConnectorException {
+        WebClient.RequestBodyUriSpec request = prepareRequest(HttpMethod.POST, connector, true);
+
+        processRequest(r -> r.uri(connector.getUrl() + NOTIFICATION_INSTANCE_IDENTIFIED_SEND_CONTEXT, uuid)
+                .retrieve()
+                .toEntity(Void.class)
+                .block()
+                .getBody(), request, connector);
+    }
+
+    public List<DataAttribute> listMappingAttributes(ConnectorDto connector, String kind) throws ConnectorException {
+        WebClient.RequestBodyUriSpec request = prepareRequest(HttpMethod.POST, connector, true);
```

Comment:
Should be HttpMethod.GET
Can you please fix it also in connector controller interface?